packages/template-retail-react-app/app/ssr.js

```diff
@@ -34,11 +34,11 @@ const options = {
 const runtime = getRuntime()
 
 const {handler} = runtime.createHandler(options, (app) => {
-    // Set HTTP security headers
+    // Use default HTTP security headers required by PWA Kit
+    app.use(defaultSecurityHeaders)
```

Comment:
What does "default" mean in this context? 🤔  we might want to be a bit more specific and call it `storefrontPreviewSecurityHeaders`(prob also a bad name) ?

Reply:
By "default" I just mean "mandatory for PWA Kit to work". So it's not _just_ Storefront Preview, even though that was the motivation for the change. And they're not _strictly_ required if a project is using `pwa-kit-runtime` for something other than the retail react app.

Naming things is hard. 🤔

Reply:
If I'm a developer, I would want to know the benefit of opting in. 

The answer is: you can't use storefront preview on runtime admin... <- this might not be clear to end users.

Reply:
wait... i think i might have misunderstanding...

if it is "mandatory for PWA Kit to work", why do we make it an opt-in?

Reply:
Because some users don't want to have _any_ CSP header, and @johnboxall said we should make it opt-in, rather than opt-out.

Reply:
On naming: https://chat.openai.com/share/d5a18a26-08b3-47ea-a1ac-952dee512285

I don't have strong feelings, but the name should communicate that a) it is a express.js middleware [can also be implied from import] b) it sets security related headers required for using the PWA Kit

Yes, folks should be allowed to *opt into* these headers – LWR sites as an example don't want the PWA Kit CSP.

Also, some customers struggle to create a CSP. They're hard. Sometimes letting them go is the only way they can truly be free.

Reply:
My vote is on adding 'pwakit' to the name, like `defaultPwaKitSecurityHeaders`, so that it's more distinguishable. Because `helmet` also has a similar concept of defaults (see `useDefaults` on line 43).

Reply:
+1 agree `defaultPwaKitSecurityHeaders` is preferable